src/elements/select/select.visual.spec.ts

```diff
@@ -0,0 +1,251 @@
+    withEllipsis: false,
+    value: undefined as string | string[] | undefined,
+    multiple: false,
+    disabled: false,
+    required: false,
+    readonly: false,
+  };
+
+  const createOptions = (
+    disableOption: boolean,
+    group: string | boolean,
+    selectValue: string | string[] | undefined = undefined,
+  ): TemplateResult[] => {
+    return new Array(5).fill(null).map((_, i) => {
+      const value = group ? `Option ${i + 1} ${' - ' + group}` : `Option ${i + 1}`;
+      const selected = Array.isArray(selectValue)
+        ? selectValue.includes(value)
+        : selectValue === value;
+      return html`
+        <sbb-option value=${value} ?disabled=${disableOption && i < 2} ?selected=${selected}>
+          ${value}
+        </sbb-option>
+      `;
+    });
+  };
+
+  const createOptionsGroup = (
+    disableOption: boolean,
+    disableGroup: boolean,
+  ): TemplateResult => html`
+    <sbb-optgroup label="Group 1" ?disabled=${disableGroup}>
+      ${createOptions(disableOption, '1')}
+    </sbb-optgroup>
+    <sbb-optgroup label="Group 2"> ${createOptions(disableOption, '2')} </sbb-optgroup>
+  `;
+
+  const template = ({
+    borderless,
+    negative,
+    floatingLabel,
+    disableOption,
+    withOptionGroup,
+    disableGroup,
+    withEllipsis,
+    ...args
+  }: typeof defaultArgs): TemplateResult => {
+    if (args.multiple && args.value) {
+      args.value = [args.value as string];
+    }
+    return html`
+      <div>
+        <sbb-form-field
+          ?borderless=${borderless}
+          ?negative=${negative}
+          ?floating-label=${floatingLabel}
+        >
+          <label>Select</label>
+          <sbb-select
+            value=${args.value || nothing}
+            ?multiple=${args.multiple}
+            ?disabled=${args.disabled}
+            ?required=${args.required}
+            ?readonly=${args.readonly}
+            class=${args.required ? 'sbb-invalid' : nothing}
+          >
+            ${withEllipsis
+              ? html` <sbb-option value=${valueEllipsis} selected="">
+                  ${valueEllipsis}
+                </sbb-option>`
+              : nothing}
+            ${withOptionGroup
+              ? createOptionsGroup(disableOption, disableGroup)
+              : createOptions(disableOption, false, args.value)}
+          </sbb-select>
+          ${args.required ? html`<sbb-form-error>Error</sbb-form-error>` : nothing}
+        </sbb-form-field>
+      </div>
+    `;
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 400 }, () => {
+    for (const negative of [false, true]) {
+      for (const visualDiffState of [visualDiffDefault, visualDiffFocus]) {
+        it(
+          `state=above negative=${negative} ${visualDiffState.name}`,
+          visualDiffState.with(async (setup) => {
+            await setup.withFixture(
+              html`
+                <div style="position: absolute; inset-block-end: 2rem; width: calc(100% - 4rem);">
```

Comment:
Did you try without width information? I think it causes horizontal problems: https://lyne-components-visual-regression-diff-pr2878.app.sbb.ch/compare/sbb-select/state=above_negative=false_default.

Reply:
I removed as a test, in local the form-field was a bit shrinked, let's see :)

Reply:
should be ok I think.